journal.py

```diff
@@ -165,22 +198,32 @@ def detail_view(request):
         }
     }
 
+    if 'HTTP_X_REQUESTED_WITH' in request.environ:
+        to_render['entry']['created'] = entry.created.isoformat()
```

Comment:
given the `__json__` method you have on your entry class above, you should be able simply to return the entry directly here, and have it render to the same basic json object you have here.